backend/src/build-system/monitor.ts

```
@@ -103,81 +83,58 @@ export class BuildMonitor {
+  ): Promise<void> {
     const inputLength = input.length;
     const value = {
       step,
       input: inputLength,
-      output: encoder.encode(output).length,
+      output: gpt3Encoder.encode(output).length,
       generateDuration,
     };
-    if (!this.timeRecorders.has(id)) {
-      this.timeRecorders.set(id, []);
+    if (!this.timeRecorders.has(name)) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)